README.md

```diff
@@ -22,7 +22,7 @@ package with compatible version numbers for stable releases.
 - PHP 5.6+ or PHP 7+
 - [Xdebug][3] or [phpdbg][4] extension enabled (PHP 7+ is required for code
   generation to work with [phpdbg][4]).
-- [PhpSpec v3][2]
+- [PhpSpec v3 or v4][2]
```

Comment:
If `composer.json` file has this change:
```
-        "phpspec/phpspec": "~3.0",
+        "phpspec/phpspec": "^4.0",
```
Then why do we need to keep `PhpSpec v3`?

Reply:
Looks like a good remark, I don't think we need this anymore

Reply:
The `composer.json` only refers to the **latest version**, which only supports phpspec `4.0`. Composer will be able to pick up an older version of the package appropriately. For now, as long as PhpSpec3 is still used in various projects, I believe it is correct to note that extension supports both versions of PhpSpec.